modules/fvad/fvad.c

```diff
@@ -0,0 +1,365 @@
+/**
+ * @file fvad.c  Voice Activity Detection
+ *
+ * Uses libfvad from https://github.com/dpirch/libfvad
+ *
+ * Copyright (C) 2023 Lars Immisch
+ */
+#include <string.h>
+#include <stdlib.h>
+#include <re.h>
+#include <rem.h>
+#include <baresip.h>
+#include <fvad.h>
+#include "fvad.h"
```

Comment:
A name collision? You don't need the header file. Remove it!

Reply:
Done (I'd rather have one, but I dislike the name-collision too)